src/components/editor/toolbar/Toolbar.js

```diff
@@ -1,27 +1,8 @@
 import React, { Component } from "react";
-import Icon from "react-icons-kit";
 
 class Toolbar extends Component {
-  getButtonsList() {
-    let buttons = [];
-
-    for (let key in this.props.buttons) {
-      buttons.push(
-        <button
-          key={key}
-          onPointerDown={this.props.buttons[key].onPointerDown}
-          className="tooltip-icon-button"
-        >
-          <Icon icon={this.props.buttons[key].icon} />
-        </button>
-      );
-    }
-
-    return buttons;
-  }
-
   render() {
-    return <div className="format-toolbar">{this.getButtonsList()}</div>;
+    return <div className="format-toolbar" {...this.props} />;
```

Comment:
what is in props i don't get it

Reply:
it contains the children so all the toolbar buttons.
it's the same as doing `<div className="format-toolbar" children={this.props.children} />;`